### PR TITLE
adds ACLK DISABLE_CLOUD to -W buildinfo

### PR DESCRIPTION
**Pull request by underhood**
##### Summary
I just noticed #9913 from @Ferroin got merged yesterday.

This PR improves by adding better info if ACLK was disabled by user request

##### Test Plan
build with `--disable-cloud`
build with ACLK

@Ferroin good idea to bring this feature as it allows us to diagnose issues that users might have much more easily.

### CODE REVIEW

File: daemon/buildinfo.c [hunk 1]
```diff
@@ -181,6 +181,14 @@ void print_build_info(void) {
     printf("    dbengine:                %s\n", FEAT_DBENGINE);
     printf("    Native HTTPS:            %s\n", FEAT_NATIVE_HTTPS);
     printf("    Netdata Cloud:           %s\n", FEAT_CLOUD);
+#ifndef ENABLE_ACLK
+#ifdef DISABLE_CLOUD
+    printf("        Cloud Disabled By User Request (e.g. installer flag --disable-cloud)\n");
+#else
+    printf("        Usable Libwebsockets:       %s\n", FEAT_LWS);
+    printf("        Usable Mosquitto:           %s\n", FEAT_MOSQUITTO);
```
Comment (Ferroin) [inline daemon/buildinfo.c:189]: Strictly speaking, we shouldn't need this block here, as we have both LWS and mosquitto listed in the libaries.

I do agree that we should have something special to indicate that the user explicitly disabled cloud support though.

Comment (underhood) [reply daemon/buildinfo.c:189]: yep you are right, I wanted to have it in one place but I can remove

Comment (underhood) [reply daemon/buildinfo.c:189]: I tried to keep it together also in configure script:
so you have all you need near the line that says `can't build ACLK` and no need to scroll and guess..
but you are right this is not user facing thing but more of a troubleshooting so I will remove it

Comment (underhood) [reply daemon/buildinfo.c]: changed it to your wishes

**Review by mfundul — APPROVED** [review]

**Review by Ferroin — APPROVED** [review]

**Pull request merged**